src/SwedbankPay.Sdk/PaymentInstruments/Invoice/ICancelTransaction.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    /// <summary>
+    /// Perform the create-cancellation operation to cancel a previously authorized or partially captured invoice payment.
+    /// </summary>
+    public interface ICancelTransaction

```

Comment:
Because it's not a response class and is modelled after the documentation: [https://developer.swedbankpay.com/payment-instruments/invoice/after-payment#create-cancel-transaction](https://developer.swedbankpay.com/payment-instruments/invoice/after-payment#create-cancel-transaction)

Reply:
If it's not a response class, why is it modelled with an interface? Is this part of the code actually tested so we know it is mapped properly to and from the request and response?

Reply:
It's used in the `InvoicePaymentCancelRequest` and is modelled after the API. I can check if there is a test for this case.

Reply:
Ok, it's not a point for `Request` classes to use interfaces, tbh. Only if they are 1:1 with the interfaces used in `Response` they can be reused, but if they are different, I see no reason to use interfaces in `Request` classes at all.

Reply:
The `ITransaction` is used many places, but not here as it does not fit. But I can delete the `ICancelTransaction`